doc/bootloader-building.rst

```diff
@@ -46,6 +46,11 @@ By setting the environment variable ``PYINSTALLER_COMPILE_BOOTLOADER``
 the pip_ setup will attempt to build the bootloader for your platform, even
 if it is already present.
 
+Doing so would execute the command ``python ./waf configure all`` upon installation.
+
+  You can pass additional arguements to the build process by setting the 
+  ``PYINSTALLER_BOOTLOADER_WAF_ARGS`` environment variable.
+
```

Comment:
Can we merge this new bit with the paragraph above so that it's just one paragraph? And fix that extraneous `e` in `arguments`?

Reply:
No, I meant squash all of it. i.e. Delete the blank line 48. And remove the indentation from lines 49 and 50 - it shows up literally in the rendered docs.

Reply:
> No, I meant squash all of it. i.e. Delete the blank line 48. And remove the indentation from lines 49 and 50 - it shows up literally in the rendered docs.

about the indentation, I thought it would be a side note. Should I instead add it to the main paragraphs?

Reply:
At least to me, it's no more of a side note than `PYINSTALLER_COMPILE_BOOTLOADER` is. And [it just looks weird](https://pyinstaller--7796.org.readthedocs.build/en/7796/bootloader-building.html#building-the-bootloader) indented.